app/helpers/layout_helper.rb

```diff
@@ -54,40 +54,34 @@ def popover(title, msg, options = {})
   end
 
   def will_paginate(collection = nil, options = {})
-    options.merge!(:class=>"col-md-7")
```

Comment:
can we move all of the pagination related core into app/helpers/pagination_helper.rb ?